### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by kambereBr**
## In this pull request, the below feature were implemented:

- Event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implemented a function for the Clear all completed button using the filter() method.
- Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads meaning that any changes made to the list are preserved.

### CODE REVIEW

File: src/index.js
Comment (fobadara) [inline src/index.js:1]: [REQUIRED] The code that handles methods related to the status updates (`completed`: `true` / `false`) should be imported as a module. Splitting your files into modules help with separation of concerns and make your code clean and readable

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @kambereBr, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Descriptive README

## Required Changes ♻️
_Check the comments under the review._
I had to include comments here because the affected file is not available in your recent changes
[REQUIRED] When I run your app it comes up but nothing works. I can't add any todos nor check if it is saved to localstorage. Please fix this 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js
Comment (kambereBr) [reply src/index.js:1]: Thanks @fobadara for your review.

I've applied your recommendations. However, I've tried to run my application with 3 different browsers (Chrome, Firefox and Edge), I didn't find any issue according to the issue that you've mentioned above. **I'm able to add, edit, delete items to the list without any problem.**

Thanks !

Comment (fobadara) [reply src/index.js:1]: How do you run it

Comment (kambereBr) [reply src/index.js:1]: I've cloned it in my other computer to test if it works, 
here are steps that I followed:

1. Clone the project
2. run :  `npm install --save-dev webpack-cli web`
3. run : `npm run build`
4. run: `npm run dev`

It's running in my second computer without any problem

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @kambereBr , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done implementing the changes requested by the previous reviewer.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
Thanks @ridwanediallo 👏

**Pull request merged**